.github/dependabot.yml

```diff
@@ -0,0 +1,8 @@
+version: 2
+updates:
+  - package-ecosystem: "pip"
+    directory: "/cli"
+    schedule:
+      interval: "weekly"
+    allow:
+      - dependency-name: "etos-lib"
```

Comment:
Why isn't this the same format as in https://github.com/eiffel-community/etos-test-runner/pull/67 ?